emcc.py

```diff
@@ -3749,7 +3750,9 @@ def process_libraries(libs, lib_dirs, temp_files):
   # Find library files
   for i, lib in libs:
     logger.debug('looking for library "%s"', lib)
-    suffixes = STATICLIB_ENDINGS + DYNAMICLIB_ENDINGS
+    suffixes = list(STATICLIB_ENDINGS + DYNAMICLIB_ENDINGS)
+    if not shared.Settings.WASM_BACKEND:
+      suffixes.append('.bc')
```

Comment:
this seems unrelated, or I'm missing the connection?

Reply:
This allows `-lc` to work on the command line for  for asmjs librarys such as libc.bc.